embulk-jruby-extension/src/main/java/org/jruby/util/ParsedValues.java

```diff
@@ -0,0 +1,406 @@
+        ZONE_OFFSET_MAP.put("ekaterinburg",            18000);
+        ZONE_OFFSET_MAP.put("fiji",                    43200);
+        ZONE_OFFSET_MAP.put("fle",                      7200);
+        ZONE_OFFSET_MAP.put("greenland",              -10800);
+        ZONE_OFFSET_MAP.put("greenwich",                   0);
+        ZONE_OFFSET_MAP.put("gtb",                      7200);
+        ZONE_OFFSET_MAP.put("hawaiian",               -36000);
+        ZONE_OFFSET_MAP.put("india",                   19800);
+        ZONE_OFFSET_MAP.put("iran",                    12600);
+        ZONE_OFFSET_MAP.put("jerusalem",                7200);
+        ZONE_OFFSET_MAP.put("korea",                   32400);
+        ZONE_OFFSET_MAP.put("mexico",                 -21600);
+        ZONE_OFFSET_MAP.put("mid-atlantic",            -7200);
+        ZONE_OFFSET_MAP.put("mountain",               -25200);
+        ZONE_OFFSET_MAP.put("myanmar",                 23400);
+        ZONE_OFFSET_MAP.put("n. central asia",         21600);
+        ZONE_OFFSET_MAP.put("nepal",                   20700);
+        ZONE_OFFSET_MAP.put("new zealand",             43200);
+        ZONE_OFFSET_MAP.put("newfoundland",           -12600);
+        ZONE_OFFSET_MAP.put("north asia east",         28800);
+        ZONE_OFFSET_MAP.put("north asia",              25200);
+        ZONE_OFFSET_MAP.put("pacific sa",             -14400);
+        ZONE_OFFSET_MAP.put("pacific",                -28800);
+        ZONE_OFFSET_MAP.put("romance",                  3600);
+        ZONE_OFFSET_MAP.put("russian",                 10800);
+        ZONE_OFFSET_MAP.put("sa eastern",             -10800);
+        ZONE_OFFSET_MAP.put("sa pacific",             -18000);
+        ZONE_OFFSET_MAP.put("sa western",             -14400);
+        ZONE_OFFSET_MAP.put("samoa",                  -39600);
+        ZONE_OFFSET_MAP.put("se asia",                 25200);
+        ZONE_OFFSET_MAP.put("malay peninsula",         28800);
+        ZONE_OFFSET_MAP.put("south africa",             7200);
+        ZONE_OFFSET_MAP.put("sri lanka",               21600);
+        ZONE_OFFSET_MAP.put("taipei",                  28800);
+        ZONE_OFFSET_MAP.put("tasmania",                36000);
+        ZONE_OFFSET_MAP.put("tokyo",                   32400);
+        ZONE_OFFSET_MAP.put("tonga",                   46800);
+        ZONE_OFFSET_MAP.put("us eastern",             -18000);
+        ZONE_OFFSET_MAP.put("us mountain",            -25200);
+        ZONE_OFFSET_MAP.put("vladivostok",             36000);
+        ZONE_OFFSET_MAP.put("w. australia",            28800);
+        ZONE_OFFSET_MAP.put("w. central africa",        3600);
+        ZONE_OFFSET_MAP.put("w. europe",                3600);
+        ZONE_OFFSET_MAP.put("west asia",               18000);
+        ZONE_OFFSET_MAP.put("west pacific",            36000);
+        ZONE_OFFSET_MAP.put("yakutsk",                 32400);
+    }
```

Comment:
You created this definition based on Ruby's Date library, right? Please add source reference as a comment.